palladium/tests/test_persistence.py

```diff
@@ -390,34 +420,22 @@ def database(self, request, Database):
 
     @pytest.fixture
     def dbmodel(self, database):
-        from palladium.util import session_scope
+        return create_dbmodel(database)
 
-        model = Dummy(
-            name='mymodel',
-            __metadata__={'some': 'metadata', 'version': 1},
-            )
-
-        model_blob = gzip.compress(pickle.dumps(model), compresslevel=0)
-        chunk_size = 4
-        chunks = [model_blob[i:i + chunk_size]
-                  for i in range(0, len(model_blob), chunk_size)]
-
-        dbmodel = database.DBModel(
-            version=1,
-            chunks=[
-                database.DBModelChunk(
-                    model_version=1,
-                    blob=chunk,
-                    )
-                for chunk in chunks
-                ],
-            metadata_=json.dumps(model.__metadata__),
-            )
-
-        with session_scope(database.session) as session:
-            session.add(dbmodel)
+    @pytest.fixture
+    def DatabaseCLOB(self):
+        from palladium.persistence import DatabaseCLOB
+        return DatabaseCLOB
+        
+    @pytest.fixture
+    def database_clob(self, request, DatabaseCLOB):
```

Comment:
Would be interested to learn why changes to the `DatabaseCLOB` parts here were necessary.  It's not clear how this is related to the pooling class change.